modules/network-observability-kafka-option.adoc

```diff
@@ -5,7 +5,7 @@
 :_content-type: CONCEPT
 [id="network-observability-kafka-option_{context}"]
 = Installing Kafka (optional)
-The Kafka Operator is supported for large scale environments. You can install the Kafka Operator as link:https://access.redhat.com/documentation/en-us/red_hat_amq_streams/2.2[Red Hat AMQ Streams] from the Operator Hub, just as the Loki Operator and Network Observability Operator were installed. 
+The Kafka Operator is supported for large scale environments to use as storage for network flow data before it is enriched by the `flowlogs-pipeline`. You can install the Kafka Operator as link:https://access.redhat.com/documentation/en-us/red_hat_amq_streams/2.2[Red Hat AMQ Streams] from the Operator Hub, just as the Loki Operator and Network Observability Operator were installed. Refer to "Configuring the FlowCollector resource with Kafka" to configure Kafka as a storage option. 
```

Comment:
same remark here, this isn't for storage, this is for forwarding data in a more resilient / scalable way